src/components/colorscale/color_attributes.js

```diff
@@ -35,14 +35,19 @@ module.exports = function makeColorScaleAttributes(context) {
                 ' values are required. For example,',
                 ' `[[0, \'rgb(0,0,255)\', [1, \'rgb(255,0,0)\']]`.',
                 ' To control the bounds of the colorscale in color space,',
-                ' use `', context, '.cmin` and `', context, '.cmax`.'
+                ' use `', context, '.cmin` and `', context, '.cmax`.',
+                ' Alternatively, `colorscale` may be a palette name string',
+                ' such as `"Viridis"` or `"Greens"`.'
```

Comment:
I'm +1 for listing them all out here (there are quite a few and I'll try to think of a cleaner way of showing all of the options) but atm we don't have the list in a super accessible spot in the docs so this would probably be a relevant place to list all the options out even if it makes it a little long here.